src/egui_node.rs

```diff
@@ -9,16 +9,17 @@ use bevy::{
     ecs::world::{FromWorld, World},
     prelude::{Entity, Handle, Resource},
     render::{
+        render_asset::RenderAssetPersistencePolicy,
```

Comment:
```suggestion
        render_asset::RenderAssetUsages,
```
This has been renamed in the latest `main` branch of bevy (upstream PR [#11399](https://github.com/bevyengine/bevy/pull/11399))